package.json

```diff
@@ -33,6 +33,12 @@
   },
   "homepage": "https://github.com/fedor/node-no-config#readme",
   "dependencies": {
-    "co": "^4.6.0"
+    "babel-core": "^6.3.15",
+    "babel-loader": "^6.2.0",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "babel-tape-runner": "^2.0.1",
+    "co": "^4.6.0",
+    "tap-webpack-plugin": "^1.1.0"
```

Comment:
I think instead of *all* new dependencies there should be only `tape` (`npm install tape --save`), since no need for babel and webpack support.